subprojects/docs/src/docs/userguide/what_is_gradle.adoc

```diff
@@ -16,34 +16,67 @@
 [[what_is_gradle]]
 = What is Gradle?
 
-Gradle is an open-source https://en.wikipedia.org/wiki/Build_automation[build automation] tool flexible enough to build almost any type of software.
-Gradle makes few assumptions about what you're trying to build or how to build it.
-This makes Gradle particularly flexible.
+Gradle is a https://en.wikipedia.org/wiki/Build_automation[build automation] tool. It is available as an open-source Build Tool and an Enterprise tool.
```

Comment:
Hmmm I'm not sure about the wording here - there is a strong separation of what Gradle Enterprise is and what Gradle Build Tool is. Reading this I get the impression they are the same product offered in either open-source or enterprise modes. However they are separate products that provide complementary features.

Gradle Enterprise works with other build tools too, its not tied only to Gradle Build Tool.

Lets get input from others too.

Reply:
I made further changes, please let me know what you think.